# Equilibrium pitch angle of current h_d

float64 theta_eq